datasets/brk/dataset.json

```
@@ -2,106 +2,73 @@
   "type": "dataset",
   "id": "brk",
   "title": "brk",
-  "status": "niet_beschikbaar",
-  "version": "0.0.1",
+  "status": "beschikbaar",
+  "version": "1.0.0",
   "crs": "EPSG:28992",
   "identifier": "identificatie",
-  "owner": "Gemeente Amsterdam",
-  "creator": "bronhouder onbekend",
+  "owner": "Gemeente Amsterdam, afdeling Basisinformatie",
+  "creator": "Kadaster",
   "publisher": "Datateam Basis- en Kernregistraties",
-  "auth": "OPENBAAR",
-  "authorizationGrantor": "OIS",
+  "auth": "BRK/RO",
+  "reasonsNonPublic": "5.1 1c: Bevat persoonsgegevens",
```

Is dit niet ook "5.1 2e: Zwaarwegend belang: eerbiediging van de persoonlijke levenssfeer", ivm met financiele gegevens van burgers?
```suggestion
  "reasonsNonPublic": ["5.1 1c: Bevat persoonsgegevens", "5.1 2e: Zwaarwegend belang: eerbiediging van de persoonlijke levenssfeer"],
```